impl/maven-cli/src/main/java/org/apache/maven/cling/logging/impl/MavenSimpleConfiguration.java

```diff
@@ -37,7 +40,17 @@ public void setRootLoggerLevel(Level level) {
                     case INFO -> "info";
                     default -> "error";
                 };
-        System.setProperty("org.slf4j.simpleLogger.defaultLogLevel", value);
+
+        String defaultLogLevelKey = "org.slf4j.simpleLogger.defaultLogLevel";
+        String current = System.getProperty(defaultLogLevelKey);
+        if (current == null) {
+            System.setProperty(defaultLogLevelKey, value);
+        } else {
+            LOGGER.warn(
+                    "System property '{}' is already set to '{}' - ignoring log level from -X/-e/-q options",
```

Comment:
should we specify what the log level that's being ignored is? that is, add a third parameter?

Also, it's really easy to mix parameters up. Simple string concatenation is preferable.

Reply:
Use mvn spotless:apply and it should fix anything IntelliJ does wrong. And you can always edit in vi or whatever plain text editor you prefer.